### DISCUSSION TITLE
Platform releases don't align with Micronaut Core releases

### DISCUSSION THREAD

**Discussion by natalie-zamani** [micronaut-projects/micronaut-platform#1680] [Q&A] [answered]
We've built our project to rely on the version catalogue exposed by this project, to ensure our dependency versions all align with what Micronaut is expecting. That being said, there's a bug fix we were hoping to consume in micronaut-core 4.6.5, but there doesn't appear to be a platform release that exposes that version. 

Is the platform version catalogue still supported? Should we not be using this if we need to consume bug fixes in specific patch versions?

**Comment by dstepanov** [comment, marked as answer]
The platform release takes some time. Just because there is a Micronaut Core release doesn't mean there will be an immediate platform release. You can force a new Micronaut Core dependency and remove it after the platform release is available.
